etc/docker/dev/rucio.cfg

```diff
@@ -173,3 +173,6 @@ support_rucio = https://github.com/rucio/rucio/issues/
 
 [credentials]
 gcs = /opt/rucio/etc/google-cloud-storage-test.json
+
+[undertaker]
+expire_rules_locks_size = 10000
```

Comment:
I would suggest to add this configuration parameter in the configuration table instead.

Reply:
alright, I am pushing the new code after the PR #1950 got merged